index.js

```diff
@@ -1,8 +1,17 @@
 'use strict';
+const Funnel = require('broccoli-funnel');
+const mergeTrees = require('broccoli-merge-trees');
 
 module.exports = {
   name: require('./package').name,
 
+  treeForPublic(tree) {
+    var assetsTree = new Funnel('tests/dummy/public');
```

Comment:
shouldn't this be: 

```suggestion
    var assetsTree = new Funnel('public');
```

🤔

Reply:
I mean `tests/dummy/public` will already be part of the build, we need this `treeForPublic()` so that the app/addon consuming this will have the icons 🤔

Reply:
since `field-guide-default-template` will be consumed by another addon, there won't be any folder named `public` in the consuming addon. Hence `new Funnel('public')` will throw an error. If this an app, this will work though.

Reply:
so the funnel should be accessing the files relative to **this** addon, not the consuming addon 🤔 I think we might need to use `join()` and `__dirname` for this to work